Makefile

```diff
@@ -34,6 +34,31 @@ MK_DIR :=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 OS_BUILDER ?= $(MK_DIR)/scripts/osbuilder.sh
 
 
+# Installation variables
+DESTDIR :=
+SYSCONFDIR := /etc
+CONFIG_FILE = configuration.toml
+DESTDIR :=
+DEFAULTSDIR := /usr/share/defaults
+CCDIR := clear-containers
+IMAGE := $(WORKDIR)/container.img
+VMLINUX := $(WORKDIR)/vmlinux.container
+VMLINUZ := $(WORKDIR)/vmlinuz.container
+
+DESTCONFDIR := $(abspath $(DESTDIR)/$(DEFAULTSDIR)/$(CCDIR))
+DESTSYSCONFDIR := $(abspath $(DESTDIR)/$(SYSCONFDIR)/$(CCDIR))
+
+# Main configuration file location for stateless systems
+DESTCONFIG := $(abspath $(DESTCONFDIR)/$(CONFIG_FILE))
+
+# Secondary configuration file location. Note that this takes precedence
+# over DESTCONFIG.
+DESTSYSCONFIG := $(abspath $(DESTSYSCONFDIR)/$(CONFIG_FILE))
+
+IMAGE_DEST := $(abspath $(DESTSYSCONFDIR)/builder-containers.img)
```

Comment:
Is there any particular reason not to make the image file  `osbuilder-containers.img` (or even just `osbuilder.img` maybe) for consistency with the kernel files below?